protobuf-kafkaconnect-converter/src/main/java/com/amazonaws/services/schemaregistry/kafkaconnect/protobuf/toconnectschema/ProtobufSchemaToConnectSchemaConverter.java

```diff
@@ -122,12 +135,14 @@ private SchemaBuilder toConnectSchemaBuilderForField(final Descriptors.FieldDesc
         //We add metadata to Connect schema to store the original type used.
         if (TYPES_TO_ADD_METADATA.contains(protobufType)) {
             schemaBuilder.parameter(PROTOBUF_TYPE, protobufType.name().toUpperCase());
-        } else if (protobufType.equals(ENUM)) { //ENUM case; storing ENUM data as metadata to avoid being lost in translation.
+        }
+
+        if (protobufType.equals(ENUM)) { //ENUM case; storing ENUM data as metadata to avoid being lost in translation.
             schemaBuilder.parameter(PROTOBUF_TYPE, PROTOBUF_ENUM_TYPE);
             for (Descriptors.EnumValueDescriptor enumValueDescriptor: fieldDescriptor.getEnumType().getValues()) { //iterating through the values of the Enum to store each one
                 schemaBuilder.parameter(PROTOBUF_ENUM_VALUE + enumValueDescriptor.getName(), String.valueOf(enumValueDescriptor.getNumber()));
             }
-            schemaBuilder.parameter(PROTOBUF_ENUM_NAME, fieldDescriptor.getName());
+            schemaBuilder.parameter("ENUM_NAME", fieldDescriptor.getName());
```

Comment:
Still seeing some enum changes. Is this reverting back your previous change? We would prefer constant values.